src/feature/articles/likeDislikeArticles/__tests__/LikeDislikeComponent.test.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { LikeDislike } from '../LikeDislikeComponent';
+
+const renderLikesAndDislikes = args => {
+  const initialProps = {
+    slug: '',
+    likeArticle: () => {},
+    dislikeArticle: () => {},
+    handleLike: () => {},
+    handleDisike: () => {},
+    article: {
+      article: {}
+    },
+    history: {
+      push: jest.fn()
+    }
+  };
+  const props = { ...initialProps, ...args };
+  return shallow(<LikeDislike {...props} />);
+};
+
+describe('Get All Articles Components tests', () => {
```

Comment:
@Jkadhuwa the first parameter of the describe block is the name of the component under test. The component under test is LikeDislikeComponent, not GetAllArticlesComponent.